app/controllers/concerns/hal_actions.rb

```diff
@@ -139,6 +139,25 @@ def index_options
     o
   end
 
+  private
+
+  # TODO: Remove this method when we upgrade roar-rails
+  # Background: https://github.com/apotonick/roar-rails/blob/a109b40/lib/roar/rails/controller_additions.rb#L27
+  def consume_with_content_type!(model, options = {})
+    type = request.content_type
+    format = Mime::Type.lookup(type).try(:symbol)
+
+    if format.blank?
+      raise UnsupportedMediaType.new("Cannot consume unregistered media type '#{type.inspect}'")
```

Comment:
This class is raising errors: `NameError: uninitialized constant HalActions::UnsupportedMediaType`